[ES|QL] docs for subquery (#145137)
elastic/elasticsearch: docs/reference/query-languages/esql/esql-multi.md

> {{esql}} allows you to query across multiple indices or clusters. Learn more in the following sections:

Reviewer: ```suggestion
{{esql}} allows you to query and combine data from multiple sources. Learn more in the following sections:
```
